attributes/default.rb

```diff
@@ -18,7 +18,7 @@
   default['repose']['install_opts'] = '-o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --force-yes'
 end
 
-default['repose']['version'] = '7.3.6.0'
+default['repose']['version'] = '8.7.3.0'
```

Comment:
What's strange is that our public API endpoints report Repose 8.0.1.0 is current in use:

```
✗ curl -I  https://monitoring.api.rackspacecloud.com/v1.0
HTTP/1.1 401 Unauthorized
Date: Fri, 16 Feb 2018 14:34:43 GMT
Server: Jetty(9.2.z-SNAPSHOT)
Via: 1.1 Repose (Repose/8.0.1.0)
WWW-Authenticate: Keystone uri=https://identity.api.rackspacecloud.com.
x-trans-id: eyJyZXF1ZXN0SWQiOiI4ZWMyYzc4YS03NmI5LTQ4NWEtODBiZi1lZDVmMDUyNTU4YWYiLCJvcmlnaW4iOm51bGx9
X-LB: api0.dfw1.prod.cm.k1k.me
```